app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarWebView.java

```diff
@@ -0,0 +1,89 @@
+package net.gsantner.markor.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.View;
+import android.webkit.WebView;
+
+public class DraggableScrollbarWebView extends WebView {
+
+    private boolean _isFastScrolling = false;
+    private boolean _fastScrollEnabled = true;
+    private boolean _ltr = true;
+    private int _thumbHeight;
+
+    public DraggableScrollbarWebView(Context context) {
+        super(context);
+    }
+
+    public DraggableScrollbarWebView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public DraggableScrollbarWebView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    @Override
+    public boolean onInterceptTouchEvent(MotionEvent ev) {
+        if (!_fastScrollEnabled) {
+            return super.onInterceptTouchEvent(ev);
+        }
+        if (_isFastScrolling) {
+            return true;
+        }
+        if (ev.getActionMasked() == MotionEvent.ACTION_DOWN &&
+                (_ltr && getWidth() - getVerticalScrollbarWidth() < ev.getX()
+                        || !_ltr && getVerticalScrollbarWidth() > ev.getX())) {
```

Comment:
this looks error-prone.   better: `|| (!_ltr && getVert... > X`. This currently looks to me like it may return un-intentionally true/false.

Reply:
Ok, I've made made changes in both classes.